src/main/java/org/betonquest/betonquest/compatibility/holograms/HologramLoop.java

```diff
@@ -90,6 +92,8 @@ private HologramWrapper initializeHolograms(final int defaultInterval, final Que
         } catch (final NumberFormatException e) {
             throw new InstructionParseException("Could not parse check interval", e);
         }
+        final int maxRange = parseMaxRange(pack, section);
+
```

Comment:
You need to do it like the way i suggested it. Only if you keep the VariableNumber til the end where you need the int, the variable it resolved correctly, as now the variable is only resolved once at the beginning, and a change in the variable will never have an effect. And then you method parseMaxRange is not really helpful